src/main/webapp/WEB-INF/tags/instructor/course/courseEditInstructorListPanelBody.tag

```diff
@@ -14,83 +14,112 @@
+                                    this course.</span>
+                            </p>
+                        </c:otherwise>
+                    </c:choose>
                 </div>
-            </c:if>
-        
+            </div>
+
             <div class="form-group">
                 <label class="col-sm-3 control-label">Name:</label>
                 <div class="col-sm-9">
-                    <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_NAME%>" 
-                            id="<%=Const.ParamsNames.INSTRUCTOR_NAME%>${instructorPanel.index}" value="${instructorPanel.instructor.name}" 
-                            data-toggle="tooltip" data-placement="top" title="Enter the name of the instructor." 
-                            maxlength="<%=FieldValidator.PERSON_NAME_MAX_LENGTH%>" tabindex="4" disabled>
+                    <input class="form-control" type="text"
+                        name="<%=Const.ParamsNames.INSTRUCTOR_NAME%>"
+                        id="<%=Const.ParamsNames.INSTRUCTOR_NAME%>${instructorPanel.index}"
+                        value="${instructorPanel.instructor.name}"
+                        data-toggle="tooltip" data-placement="top"
+                        title="Enter the name of the instructor."
+                        maxlength="<%=FieldValidator.PERSON_NAME_MAX_LENGTH%>"
+                        tabindex="4" disabled>
                 </div>
             </div>
-        
+
             <div class="form-group">
                 <label class="col-sm-3 control-label">Email:</label>
                 <div class="col-sm-9">
-                    <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_EMAIL%>" 
-                            id="<%=Const.ParamsNames.INSTRUCTOR_EMAIL%>${instructorPanel.index}" value="${instructorPanel.instructor.email}"
-                            data-toggle="tooltip" data-placement="top" title="Enter the Email of the instructor." 
-                            maxlength="<%=FieldValidator.EMAIL_MAX_LENGTH%>" tabindex="5" disabled
-                            <c:if test="${empty instructorPanel.instructor.googleId}">
+                    <input class="form-control" type="text"
+                        name="<%=Const.ParamsNames.INSTRUCTOR_EMAIL%>"
+                        id="<%=Const.ParamsNames.INSTRUCTOR_EMAIL%>${instructorPanel.index}"
+                        value="${instructorPanel.instructor.email}"
+                        data-toggle="tooltip" data-placement="top"
+                        title="Enter the Email of the instructor."
+                        maxlength="<%=FieldValidator.EMAIL_MAX_LENGTH%>"
+                        tabindex="5" disabled
+                        <c:if test="${empty instructorPanel.instructor.googleId}">
                                 readonly
-                            </c:if> >
+                            </c:if>>
```

Comment:
Indentation for the `</c:if>>` seems off, it should be in line with the `<c:if>` earlier